Documentation/contributing.rst

```diff
@@ -911,6 +911,9 @@ Steps to release
     See the ``README`` in the ``contrib/release`` directory for more information.
 11. Create a GitHub release and include the release notes as well as links to
     the binaries.
+12. Update the release branch on
+    `Jenkins <https://jenkins.cilium.io/view/all/job/cilium/>`__ to be tested on
```

Comment:
Link is incorrect; it points to the Bash tests.

Reply:
Would be helpful to point to the nightly job that runs for the release branch, as well as the job that runs for each commit.